roles/nova-data/tasks/monitoring.yml

```diff
@@ -13,6 +13,7 @@
   sensu_metrics_check: name=check-cinder-sessions plugin=check-cinder-sessions.py
                        interval=10
   notify: restart sensu-client
+  when: "'rbd_volumes' not in cinder.enabled_backends"
```

Comment:
Problem is it's not a list, it's a string of comma-separated values. :(

`enabled_backends=lvmdriver-1,lvmdriver-2`